models/runs/filter-runs.mdx

```diff
@@ -19,60 +16,37 @@ Filter runs based on their tags with the filter button.
 2. Select the **Filter** button, which looks like a funnel, at the top of the runs table.
 3. From left to right, select `"Tags"` from the dropdown menu, select a logic operator, and select a filter search value.
 
+<Frame>
+    <img src="/images/app_ui/filter_runs.gif" alt="Filter runs by tags"  />
```

Comment:
Curious why the image is formatted this way. Could use a Markdown image and could set the alt text as a caption on the frame. https://www.mintlify.com/docs/components/frames#captions